<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>EqualizerDialogClass</class>
 <widget class="QWidget" name="EqualizerDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>equalizerDialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>90</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title" >
    <string>Mesh/Histogram values</string>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>301</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:MS Shell Dlg; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Min Q:&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="meshMinQlineEdit" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="histoMinQlineEdit" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:MS Shell Dlg; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Max Q:&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="meshMaxQlineEdit" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="histoMaxQlineEdit" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title" >
    <string>Histogram Parameters</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="minimumSize" >
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:MS Shell Dlg; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Percentile&lt;/p>&lt;/body>&lt;/html></string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="percentileSpinbox" >
       <property name="minimumSize" >
        <size>
         <width>65</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>65</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="prefix" >
        <string>1/</string>
       </property>
       <property name="maximum" >
        <number>98</number>
       </property>
       <property name="minimum" >
        <number>2</number>
       </property>
       <property name="value" >
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>51</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="minimumSize" >
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:MS Shell Dlg; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Range&lt;/p>&lt;/body>&lt;/html></string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="rangeSpinbox" >
       <property name="minimumSize" >
        <size>
         <width>65</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>65</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum" >
        <number>10000000</number>
       </property>
       <property name="minimum" >
        <number>100</number>
       </property>
       <property name="singleStep" >
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>200</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton" >
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="applyButton" >
      <property name="text" >
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
